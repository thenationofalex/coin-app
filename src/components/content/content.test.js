import React, {Component} from 'react'
import {shallow, mount} from 'enzyme'
import toJson from 'enzyme-to-json'

import Content from './index'
 
test('Content renders', () => {
  const wrapper = shallow(
    <Content>
      <h1>Some content</h1>
    </Content>
  );
 
  expect(toJson(wrapper)).toMatchSnapshot()
});

test('Content children is rendered', () => {
  const wrapper = mount(
    <Content>
      <h1>Some content</h1>
    </Content>
  );
 
  expect(wrapper.text()).toEqual('Some content')
});

