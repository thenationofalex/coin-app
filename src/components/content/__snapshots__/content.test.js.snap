// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content renders 1`] = `
<Styled(section)>
  <h1>
    Some content
  </h1>
</Styled(section)>
`;
